src/Views/listings/view.phtml

```diff
@@ -7,13 +13,13 @@ $title = 'listing';
         <img class="mainImage h-100 w-100" src="" alt="">
     </div>
     <div class="view-item-description listing-grid-design">
-        <h6>Seller name, Location</h6>
+        <h6><?= $listing->getUser()->getFullName() ?>, <?= $listing->getUser()->getOffice()->getAddress() ?></h6>
         <div>
-            <h4>Title</h4>
+            <h4><?= $listing->getListingName() ?></h4>
         </div>
     </div>
     <div class="view-interact-item listing-grid-design">
-        <h6>Sold for £PRICE</h6>
+        <h6>Sold for £<?=$listing->getPrice() ?></h6>
```

Comment:
put a space at the start and end within php tags e.g. <?= $name ?> instead of <?=$name?>